src/components/homeComponents/ExchangeDetails.js

```diff
@@ -32,14 +33,25 @@ const ExchangeDetails = () => {
           onChange={(e) => setSearch(e.target.value)}
         />
       </form>
-      <div className="d-flex f-end details">
+      <div className="d-flex info">
+        <div className="md-6 d-flex mid-center">
+          <img className="base-img" src={flag} alt="currency-flag" />
+        </div>
+        <div className="md-6 d-flex mid-center base-title">
+          <div className="md-6">
+            <h2>{id}</h2>
+            <div>1.000</div>
+          </div>
+        </div>
+      </div>
+      {/* <div className="d-flex f-end details">
         <div className="md-6 x-title d-flex f-end">
           <div>
             <h2>{id}</h2>
             <div>1.000</div>
           </div>
         </div>
-      </div>
+      </div> */}
```

Comment:
Great job @abiola-arobieke 🤝
- [ ] Kindly remove commented code from your project as this is regarded as bad practice.